src/Gang/ui/RecruitButton.tsx

```diff
@@ -15,24 +14,29 @@ interface IProps {
 export function RecruitButton(props: IProps): React.ReactElement {
   const gang = useGang();
   const [open, setOpen] = useState(false);
-  if (gang.members.length >= GangConstants.MaximumGangMembers) {
-    return <></>;
-  }
 
   if (!gang.canRecruitMember()) {
     const respect = gang.getRespectNeededToRecruitMember();
     return (
       <Box display="flex" alignItems="center" sx={{ mx: 1 }}>
         <Button disabled>Recruit Gang Member</Button>
-        <Typography sx={{ ml: 1 }}>{formatRespect(respect)} respect needed to recruit next member</Typography>
+        {gang.getRecruitsAvailable() === -1 && (
+          <Typography sx={{ ml: 1 }}>Maximum gang members already recruited</Typography>
+        )}
+        {gang.getRecruitsAvailable() !== -1 && (
+          <Typography sx={{ ml: 1 }}>{formatRespect(respect)} respect needed to recruit next member</Typography>
+        )}
```

Comment:
If changing getRecruitsAvailable to return 0 when you're at max members and getRespectNeededToRecruitMember changed to return Infinity, then respect needs to be used as the condition here.
Also a ternary works better here than a condition + anticondition (formatting on suggestion might be wrong, typed on github and not in an IDE)

```suggestion
        {respect === Infinity ? (
          <Typography sx={{ ml: 1 }}>Maximum gang members already recruited</Typography>
        ) : (
          <Typography sx={{ ml: 1 }}>{formatRespect(respect)} respect needed to recruit next member</Typography>
        )}
```